makefile.shade

```diff
@@ -203,6 +204,32 @@ var buildTarget = "compile"
         }
     }
 
+#cache-coherence
+    @{
+        var coherenceCacheDir = Path.Combine(BASE_DIR, Environment.GetEnvironmentVariable("COHERENCE_CACHE_DIR") ?? ".coherence-cache");
+        
+        if (!Directory.Exists(coherenceCacheDir))
+        {
+            var dropsShare = Environment.GetEnvironmentVariable("ASPNETCI_DROPS_SHARE");
+
+            if (!string.IsNullOrWhiteSpace(dropsShare))
+            {
+                var coherenceShare = Path.Combine(dropsShare, "Coherence", BUILD_BRANCH);
```

Comment:
I'm wondering if this can fail if you get a partially pushed coherence... Maybe coherence should drop a file as the final step and we should look only at the folders that have that file